### PR TITLE
HCF 912: Property report

### PR DESCRIPTION
**Pull request by andreas-kupries**

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -28,6 +28,7 @@ var (
 	flagConfiggin      string
 	flagLightOpinions  string
 	flagDarkOpinions   string
+	flagJSON           bool
```
Comment (viovanov) [inline cmd/root.go]: let's use an `--output` flag
It can have a default value of `human` and could also be `json` or `yaml` perhaps?

**Comment by viovanov** [comment]
Can you please add a sample of the output?

File: cmd/root.go (hunk 1 above)
Comment (andreas-kupries) [inline cmd/root.go]: Sure. Note however that currently *only* `show properties` reacts to this flag in any way, shape, or form.

**Comment by andreas-kupries** [comment]
Sample of the output ... Attached to https://jira.hpcloud.net/browse/HCF-912 now.
Name is `fissile-show-properties.json`. Needs to run through `jq` to be nicely formatted.

File: cmd/root.go (hunk 1 above)
Comment (andreas-kupries) [inline cmd/root.go]: Done with commit 46881bfff4c6d0b74fced5e03fa0dd0456d43580.
Example of yaml output attached to the ticket.

File: app/fissile.go [hunk 2]
```diff
@@ -207,6 +210,78 @@ func (f *Fissile) ListJobs() error {
...
+
+			for _, property := range job.Properties {
+				f.UI.Printf("\t%s: %s\n", color.YellowString(property.Name),
+					job.Description)
+			}
+		}
+	}
+}
+
+type props map[string]map[string][]string
```
Comment (viovanov) [inline app/fissile.go]: only used twice - i don't think we need to define this for the whole package

File: cmd/root.go [hunk 3]
```diff
@@ -186,6 +188,13 @@ func init() {
 		"Path to a BOSH deployment manifest file that contains properties that should not have opinionated defaults.",
 	)
 
+	RootCmd.PersistentFlags().StringP(
+		"output",
+		"o",
+		"human",
+		"Choose output format, one of human, json, or yaml (currently only 'show properties')",
```
Comment (viovanov) [inline cmd/root.go]: currently only _for_ ...

File: app/fissile.go (hunk 2 above)
Comment (andreas-kupries) [inline app/fissile.go]: Done with commit baaf4f2cad73091c586c519c1e2c38975f90d88f

File: cmd/root.go (hunk 3 above)
Comment (andreas-kupries) [inline cmd/root.go]: Done with commit baaf4f2cad73091c586c519c1e2c38975f90d88f

**Comment by viovanov** [comment]
👍

**Pull request merged**